src/Core/src/Hosting/LifecycleEvents/AppHostBuilderExtensions.iOS.cs

```diff
@@ -105,21 +105,30 @@ static void OnConfigureWindow(IiOSLifecycleBuilder iOS)
 		{
 			// Pre iOS 13 doesn't support scenes
 			if (!OperatingSystem.IsIOSVersionAtLeast(13))
+			{
 				return;
+			}
 
 			iOS = iOS
 				.WindowSceneDidUpdateCoordinateSpace((windowScene, _, _, _) =>
 				{
-					if (!OperatingSystem.IsIOSVersionAtLeast(13))
+					// Mac Catalyst version 16+ supports effectiveGeometry property on window scenes.
+					if (!OperatingSystem.IsIOSVersionAtLeast(13) || (OperatingSystem.IsMacCatalyst() && OperatingSystem.IsIOSVersionAtLeast(16)))
```

Comment:
Similar to the question above, can we use `IsMacCatalystVersionAtLeast` instead?